### PR TITLE
Gradle Installation

### PR DESCRIPTION
**Pull request by haroon-sheikh**
#90 - Adds installation instructions
#91 - Adds a note to make sure the version of dependencies are same.

### CODE REVIEW

File: content/configuration.md [hunk 1]
```diff
@@ -36,7 +35,7 @@ java cucumber.api.cli.Main
 
 {{% block "javascript" %}}
 ### CLI
-Cucumber.js includes an executable file to run the features. After installing Cucumber in your project, you can run it with:
+Cucumber.js includes an executable file to run the features. After installing Cucumber in project, you can run it with:
```
Comment (olleolleolle) [inline content/configuration.md]: Perhaps "in project" is a little bare?

Comment (haroon-sheikh) [reply content/configuration.md]: How does `in a project` sound? You hardly see `you\your` in docs.

File: content/installation.md [hunk 2]
```diff
@@ -5,11 +5,15 @@ polyglot: true
 
 {{% block "java" %}}
 
-Cucumber-JVM is published as several JAR files in the central Maven repository. Installation is simply a matter of adding dependencies to your build file:
+Cucumber-JVM is published as several JAR files in the central Maven repository. Installation is simply a matter of adding dependencies to build files:
```
Comment (olleolleolle) [inline content/installation.md]: Can we move towards dropping "easy" or "simple" from instructions?

Perhaps:

> Install by adding dependencies to build files:

File: content/installation.md [hunk 3]
```diff
@@ -26,6 +30,24 @@ Add the following dependency to your `pom.xml`:
...
+    testCompile 'io.cucumber:cucumber-java:{{% version "cucumberjvm" %}}'
+    testCompile 'io.cucumber:cucumber-junit:{{% version "cucumberjvm" %}}'
+}
+
+repositories {
+    mavenCentral()
+}
+```
+
+You can use now cucumber's [CLI Runner](/configuration/#configuration) to execute by adding the following `cucumber` task to `build.gradle`. See [Configuration](/configuration):
```
Comment (olleolleolle) [inline content/installation.md]: Typo: `can use now` => `can now use`

**Comment by olleolleolle** [comment]
@haroon-sheikh Hello and thank you for improving the documentation!

I made detail notes as inline comments.

**Comment by haroon-sheikh** [comment]
@olleolleolle No problems, I've pushed the fixes.

**Review by olleolleolle — APPROVED** [review]
These clarifications correct and simplify the configuration chapter.

**Comment by olleolleolle** [comment]
@haroon-sheikh Boom!

**Comment by olleolleolle** [comment]
@haroon-sheikh I think the changes were super and I hope that you were finished making them.

**Comment by haroon-sheikh** [comment]
Thanks @olleolleolle - Apologies, I had left a typo in which i've corrected in master.

File: content/configuration.md (hunk 1 above)
Comment (plaindocs) [reply content/configuration.md]: We quite often use you/your these days, in more informal docs written in active tense rather than passive.

**Comment by plaindocs** [comment]
Thanks @haroon-sheikh for the PR and speedy fixes and @olleolleolle for the review!

**Comment by brasmusson** [comment]
@haroon-sheikh To let GitHub work for you, include [keywords that close issues](https://help.github.com/articles/closing-issues-using-keywords) in the pull request description for the issues that are fixed (for instance `fixes #90` instead of just `#90` - backtics used here so GitHub does not parse the examples), then GitHub automatically close the fixed issues with the pull request is merged (now the issues #90 and #91 had to be closed manually).

**Pull request merged**